templates/overtimereport-yearly.html

```diff
@@ -32,10 +35,10 @@ <h1>Attendance for {{ .Username }}</h1>
     <tr>
         <td><a href="{{ .DetailViewLink }}">{{ .Name }}</a>{{- with .ValidationError }} ⚠️{{- end }}</td>
         <td>{{ .LeaveDays }}d</td>
-        <td class="text-end font-monospace">{{ .ExcusedHours }}</td>
-        <td class="text-end font-monospace" >{{ .WorkedHours }}</td>
-        <td class="text-end font-monospace {{ .OvertimeClassname }} ">{{ .OvertimeHours }}</td>
-        <td class="text-end font-monospace">{{ .DefinitiveBalance }}</td>
+        <td class="text-end font-monospace bold">{{ .ExcusedHours }}</td>
+        <td class="text-end font-monospace bold" >{{ .WorkedHours }}</td>
+        <td class="text-end font-monospace bold {{ .OvertimeClassname }} ">{{ .OvertimeHours }}</td>
+        <td class="text-end font-monospace bold">{{ .DefinitiveBalance }}</td>
```

Comment:
I wouldn't change the font weight in the yearly report, at all. Only the weekly/create report and monthly.
Reasoning: The overtime in the yearly report is not that relevant for the user.